chalice/tests/chalicelib/criteria/test_criteria_default.py

```diff
@@ -244,7 +244,9 @@ def test_init_state(self):
         test that all instance variables have the expected initial values
         """
         with self.subTest():
-            self.assertTrue(self.subclass.active, msg="active must be True")
+            # Checks do not have to be active but value should be set and boolean
+            #    self.assertTrue(self.subclass.active, msg="active must be True")
+            self.assertIn(self.subclass.active, [True,False], msg="active must be True")
```

Comment:
I think the `msg` doesn't match the condition. If `self.subclass.active is False`, then the test won't fail, but the `self.subclass.active` won't be `True`.

Reply:
Agreed - fixing now.

Reply:
I've rebased this to pick up the CT-692 merge